appveyor.yml

```diff
@@ -0,0 +1,29 @@
+version: 1.0.{build}
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  except:
+  - coverity_scan
+  - gh_pages
+skip_tags: true
+image:
+  - Visual Studio 2017
+configuration:
+  - Debug
+  - Release
+platform:
+  - x64
+  - Win32
+init:
+  - cmd: |
+      cmake -version
+before_build:
+  - cmd: |
+      cd /d %APPVEYOR_BUILD_FOLDER%
+      if "%PLATFORM%"=="Win32" mkdir build & cd build & cmake -G"Visual Studio 15 2017" ..
+      if "%PLATFORM%"=="x64" mkdir build & cd build & cmake -G"Visual Studio 15 2017 Win64" ..
+
+build_script:
+  - cmd: |
+      cd /d %APPVEYOR_BUILD_FOLDER%
+      MSBuild /verbosity:minimal build/Jerry.sln /p:Configuration="%CONFIGURATION%" /p:Platform="%PLATFORM%"

```

Comment:
Two questions here:
- Is the `cd` necessary? See above comment.
- Is `cmd` necessary at all or is just `project`, `verbosity`, and `parallel` enough? See previous comment with example.

Reply:
@akosthekiss no enough, I need to speicify the Platform.

Reply:
I don't think that's correct. Just tried to simplify the configuration: https://github.com/akosthekiss/jerryscript/commit/9b4cbc9ba04a3b26ad57a1d80e959ab7a2f257d0 . It works well, as expected. The AppVeyor CI passes both configuration and platform to msbuild when using the `build` approach: https://ci.appveyor.com/project/akosthekiss/jerryscript/build/1.0.7 . You are welcome to use the patch.